source/js/plugins/pixel-wrapper.js

```diff
@@ -145,9 +147,15 @@ export class PixelWrapper
             width = this.pixelInstance.core.publicInstance.getPageDimensionsAtZoomLevel(pageIndex, maxZoom).width,
             height = this.pixelInstance.core.publicInstance.getPageDimensionsAtZoomLevel(pageIndex, maxZoom).height;
 
-        // Highlight whole image for background layer
-        let rect = new Rectangle(new Point(0, 0, pageIndex), width, height, "add");
-        backgroundLayer.addShapeToLayer(rect);
+        // Add select regions to backgroundLayer
+        selectRegionLayer.shapes.forEach((shape) => {
+            let x = shape.origin.getCoordsInPage(maxZoom).x,
+                y = shape.origin.getCoordsInPage(maxZoom).y,
+                rectWidth = shape.relativeRectWidth * 2^maxZoom,
```

Comment:
Be consistent with the spacing of math operators

Reply:
Will do!

Reply:
Wait, this has been changed to `Math.pow(2, maxZoom)` already. Not sure why it's showing like so here